content/_data/changelogs/weekly.yml

```
@@ -9625,6 +9625,73 @@
   # pull: 5085 (PR title: Removed outdated browser workarounds)
   # pull: 5086 (PR title: Fixed some stylelint issues)
 
+- version: '2.270'
+  date: 2020-12-08
+  changes:
+  - type: major bug
```

```suggestion
  - type: bug
```

Does not look to be so major